.github/workflows/build-frontend.yml

```diff
@@ -14,6 +14,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Install Bowtie
+        uses: ./
+
+      - name: Generate all implementations (harness)'s metadata
+        run: |
+          bowtie info $(ls implementations/ | sed 's/^/-i /') --format file  | tee implementations.json
```

Comment:
`tee` is used for writing to multiple places, as in the example you copied from where we also want the output to go to `bowtie badges`. Here you have only one place, and can just use `>` to write directly to the file you want it to go to.